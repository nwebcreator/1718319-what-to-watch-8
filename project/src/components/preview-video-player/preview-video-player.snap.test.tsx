import { render } from '@testing-library/react';
import PreviewVideoPlayer from './preview-video-player';

describe('Component: PreviewVideoPlayer', () => {
  it('should render correctly', () => {
    const { container } = render(<PreviewVideoPlayer src='' />);
    expect(container).toMatchSnapshot();
  });
});
