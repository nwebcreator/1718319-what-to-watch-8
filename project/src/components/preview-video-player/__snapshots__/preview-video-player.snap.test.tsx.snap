// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: PreviewVideoPlayer should render correctly 1`] = `
<div>
  <video>
    <source
      src=""
    />
  </video>
</div>
`;
